syncapi/types/types.go

```diff
@@ -220,8 +223,8 @@ func NewTopologyTokenFromString(tok string) (token TopologyToken, err error) {
 		err = fmt.Errorf("token %s is not a topology token", tok)
 		return
 	}
-	if len(t.Positions) != 2 {
-		err = fmt.Errorf("token %s wrong number of values, got %d want 2", tok, len(t.Positions))
+	if len(t.Positions) < 2 {
+		err = fmt.Errorf("token %s wrong number of values, got %d want at least 2", tok, len(t.Positions))
```

Comment:
Where is the extra value coming from? That shouldn't really be allowed as Topology tokens only have 2 params.

Reply:
Don't ask me why but on occasion it seems Riot iOS tries to send us tokens that are longer than that.